bip-update-process.md

```diff
@@ -0,0 +1,516 @@
+discussion, address feedback and dissenting opinions, and, if applicable, advance adoption of their proposal within the
+Bitcoin community.
+
+The BIPs repository serves as a highly visible platform to put forward mature proposals. The repository’s visibility
+facilitates the distributed consideration of proposals by establishing the current state of the proposal. The
+repository’s versioning of the text-based BIPs retains the historical record of each proposal.
+The BIPs repository does not[^acceptance] aim to track acceptance, adoption, or community consensus on BIPs except to facilitate a
+quick overview of various BIPs’s statuses to repository visitors.
+
+This BIP replaces BIP 2 with a more well-defined and clear process, and may be amended to address future needs of the
+BIP process.
+
+## What is a BIP?
+
+### What is the significance of BIPs?
+
+Individual BIPs do not represent Bitcoin community consensus or a general recommendation for implementation. A BIP
+represents a personal recommendations by the BIP champions to the Bitcoin community. Some BIPs may be adopted by one or
+multiple implementations or even incorporated into the Bitcoin protocol.
+
+### What should be documented in a BIP?
+
+The BIPs repository collects any proposals relevant to the Bitcoin community.
+
+OR:
+
+The BIPs repository is focused on information and technology that supports the bitcoin currency.[^astroturfing]
+
+## BIP format and structure
+
+### Specification
+
+BIPs should be written in mediawiki or markdown format.
+
+Each BIP should have the following parts:
+
+* Preamble -- Headers containing metadata about the BIP (see the section [BIP header preamble](#bip-header-preamble)
+  below).
+* Abstract -- A short (~200 word) description of the technical issue being addressed.
+* Motivation -- The motivation is critical for BIPs. It should clearly explain what issue the BIP addresses, and how the
+  existing situation is inadequate to address the problem that the BIP solves.
+* Rationale -- The rationale fleshes out the specification by describing what motivated the design and why particular
+  design decisions were made. It should describe alternate designs that were considered and related work. The rationale
+  should address important objections or concerns raised during discussion.
+* Specification -- The technical specification should describe the syntax and semantics of any new feature. The
+  specification should be detailed enough to enable any Bitcoin project to create an interoperable implementation.
+* Backwards compatibility -- BIPs that introduce backwards incompatibilities must include a section describing these
+  incompatibilities and their severity. The BIP must explain how the champion proposes to deal with these
+  incompatibilities.
+* Reference implementation -- Where applicable, a reference implementation, test vectors and documentation must be
```

Comment:
Both reference implementations and test vectors might make more sense to just be auxiliary files (eg bip-0340 works this way), with a link from the document than being included in the doc itself?

Perhaps it would make sense to link to the first one or two real implementations in the preamble in a similar way to Post-History, perhaps as a pre-requisite for switching from Nominated to Active.